docs/topics/api/signing.rst

```diff
@@ -46,9 +46,11 @@ validation and you will be able to check its status.
             -g -XPUT --form 'upload=@build/my-addon.xpi'
             -H 'Authorization: JWT <jwt-token>'
 
-    :param addon-id the id for the add-on.
-    :param version: the version of the add-on.
-    :form upload: the add-on being uploaded.
+    :param addon-id: The id for the add-on.
```

Comment:
I find it odd that these are URL parts are documented as if they were POST parameters. Maybe you could make a new section for them above "Request" like "Creating a URL" or something.